app/helpers/quadicon_helper.rb

```diff
@@ -553,16 +553,26 @@ def db_for_quadicon
   def render_ext_management_system_quadicon(item, options)
     output = []
 
-    if settings(:quadicons, db_for_quadicon)
+    if true # settings(:quadicons, db_for_quadicon)
```

Comment:
@himdel im having trouble getting the setting `ems_container` to appear under quadicons.